node/utils/src/lib.rs

```diff
@@ -6,15 +6,30 @@ use std::fs::{create_dir_all, File};
 use std::io::{prelude::*, Result};
 use std::path::Path;
 
+/// Restricts permissions on a file to user-only: 0600
+#[cfg(unix)]
+pub fn set_user_perm(file: &File) -> Result<()> {
+    use log::info;
+    use std::os::unix::fs::PermissionsExt;
+
+    let mut perm = file.metadata()?.permissions();
+    perm.set_mode((libc::S_IWUSR | libc::S_IRUSR) as u32);
```

Comment:
Do you think `0o600` would be better here?

Reply:
I think using the mask like C style #defines works fine here. Write | Read looks better than `600` from a readability perspective.